controllers/grid/eventLog/EventLogGridRow.inc.php

```diff
@@ -25,11 +25,18 @@ class EventLogGridRow extends GridRow {
 	/** @var Submission **/
 	var $_submission;
 
+	/** @var boolean Is the current user assigned as an author to this submission */
+	var $_isCurrentUserAssignedAuthor;
+
 	/**
 	 * Constructor
+	 * @param Submission $submission
+	 * @param boolean $isCurrentUserAssignedAuthor Is the current user assigned
+	 *  as an author to this submission?
 	 */
-	function __construct($submission) {
+	function __construct($submission, $isCurrentUserAssignedAuthor) {
```

Comment:
should here maybe the second parameter be false per default, as above? -- both classes (this one and the above) are currently called only once, thus not so important, but...

Reply:
My thinking was that any time we're showing the event log, we need to be checking if log details should be anonymized. For that reason, I thought it best to require this be passed in, so that an implementer in a few years wouldn't just ignore it without thinking...

Reply:
:+1:  shall we then make it required in the class above?

Reply:
Yes, good thinking.